service/swagger.js

```diff
@@ -39,7 +39,8 @@ const Team = Joi.object({
   path: Joi.string(),
   users: Joi.array().items(UserRef),
   policies: Joi.array().items(PolicyRef),
-  organizationId: Joi.string()
+  organizationId: Joi.string(),
+  membersCount: Joi.number()
```

Comment:
I found also a bit confusing the the naming inconsitency: `users` vs `membersCount`: I'd should go with either `users` and `usersCount` (preferred) or `members` and `membersCOunt`